### ISSUE TITLE
_KUBE_CROSS_VERSION" in the substitution data is not matched in the template

### ISSUE THREAD

**Issue by Bubblemelon** [kubernetes/release#724]
```bash
root@a7c24c88c1b0:/release# ./gcbmgr release  release-1.15 --buildversion=v1.15.0-beta.0.425+e0677859bbcaf6

gcbmgr: BEGIN main on a7c24c88c1b0 Wed May 29 00:47:03 UTC 2019

Checking required system packages: OK
Checking /release state: OK
Checking/setting cloud tools: OK

FAILED: Job was not submitted.  Details:
ERROR: (gcloud.builds.submit) INVALID_ARGUMENT: invalid build: key
"_KUBE_CROSS_VERSION" in the substitution data is not matched in the template
```

From `gcbmgr.log`:
```bash
gcbmgr::submit_it(): ESC[1mESC[31mFAILEDESC(BESC[m: Job was not submitted.  Details:
gcbmgr::submit_it(): ERROR: (gcloud.builds.submit) INVALID_ARGUMENT: invalid build: key
"_RELEASE_TOOL_REPO" in the substitution data is not matched in the
template;key "_RELEASE_TOOL_BRANCH" in the substitution data is not matched
in the template;key "_KUBE_CROSS_VERSION" in the substitution data is not
matched in the template

```

This occurs in the [`submit_it()`](https://github.com/kubernetes/release/blob/255bd062b65c896aeb8aba7583158c6fff4cbc9d/gcbmgr#L289 ) function.

cc: @hoegaarden

**Comment by hoegaarden** [comment]
We could just extend https://github.com/kubernetes/release/blob/255bd062b65c896aeb8aba7583158c6fff4cbc9d/gcbmgr#L296 with `_KUBE_CROSS_VERSION` and remove https://github.com/kubernetes/release/blob/255bd062b65c896aeb8aba7583158c6fff4cbc9d/gcbmgr#L303

**Comment by hoegaarden** [comment]
JFYI @kubernetes/patch-release-team

**Comment by justaugustus** [comment]
Closed via #725.
/close
